app/components/card_component.rb

```diff
@@ -14,6 +14,7 @@ def card_classes_wrapper
     [
       "shadow",
       "rounded-lg",
+      "h-full",
```

Comment:
I think this should probably not be in this pr; but I am not going to fiddle with it more tonight.